assets/js/custom.js

```diff
@@ -78,32 +83,32 @@ jQuery(document).ready(function() {
         return `<div><strong>${ domain? capitalize(tableDomains): ""}</strong></div>
                 <table id="ont_table" class="table table-hover sortable">
                     <thead>
-                        <tr class="row">
-                            <th scope="col" class="ob-center">
+                        <tr>
+                            <th scope="col">
                                 <span>ID</span>
                                 <button type="button" class="btn btn-outline-default btn-xs" title="Sort by ID" data-sort="id" >
-                                    <span aria-hidden="true" class="glyphicon glyphicon-chevron-up"></span>
+                                    <i class="bi-chevron-up" aria-hidden="true"></i>
```

Comment:
This is just how the icon libraries work, the class has CSS associated with it that fills up the `<i>` element with some SVG in a magical way I don't really understand. Why it's specifically `<i>` and not `<span>` I'm not sure about, maybe because it's short?

Reply:
ok!